cmd/podman/root.go

```diff
@@ -58,7 +58,6 @@ var (
 		Long:               "Manage pods, containers and images",
 		SilenceUsage:       true,
 		SilenceErrors:      true,
-		TraverseChildren:   true,
```

Comment:
I thiiiink this might disable global flags intermixing with command-level flags, which I think we still want - `podman run -t -i --rm --log-level=debug fedora ls /` should still work

Reply:
@mheon That still works.  TraverseChildren is about parsing parent's local flags before child command.  `root` has no parents.